mappings/net/minecraft/client/render/entity/EntityRendererStrayOverlay.mapping

```diff
@@ -0,0 +1,10 @@
+CLASS none/bvz net/minecraft/client/render/entity/EntityRendererStrayOverlay
+	FIELD a OVERLAY_TEXTURE Lnone/kp;
```

Comment:
`SKIN` is currently the standard for anything that goes on a 3D model. If we decide it should be something different, we can easily search/replace all occurrences.